File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.js
function solution(my_string, num1, num2) {
    let char1 = my_string[num1];
    let char2 = my_string[num2];
    let str = my_string.split("");
    str[num1] = char2;
    str[num2] = char1;
    return str.join("");
}

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    string = ""
    
    for idx, str in enumerate(my_string):
        if idx == num1:
            string += my_string[num2]
        elif idx == num2:
            string += my_string[num1]
        else:
            string += str
            
    return string